### PR TITLE
extensions: PDFJS toolbar fixes

### PR DESCRIPTION
**Pull request by JavierDelgadoFernandez**
* Fixes the issues with the toolbar buttons updating the npm package
  to the official repository.

Signed-off-by: Javier Delgado <javier.delgado.fernandez@cern.ch>

### CODE REVIEW

**Comment by JavierDelgadoFernandez** [comment]
@lnielsen It is ready from my side.

**Comment by JavierDelgadoFernandez** [comment]
@lnielsen Rebased

File: invenio_previewer/bundles.py [hunk 1]
```diff
@@ -75,31 +75,16 @@
...
+            "pdfjs-dist": "1.4.192",
         }
     ),
-    "node_modules/pdf-viewer/ready/generic/web/compatibility.js",
-    "node_modules/pdf-viewer/ready/generic/web/l10n.js",
-    "node_modules/pdf-viewer/ready/generic/web/viewer.js",
-    "node_modules/pdf-viewer/ready/generic/build/pdf.js",
-    "node_modules/pdf-viewer/ready/generic/build/pdf.worker.js",
-    "js/pdfjs/pdf_viewer.js",
-    "js/zip/fullscreen.js",
```
Comment (lnielsen) [inline invenio_previewer/bundles.py]: ..

**Comment by JavierDelgadoFernandez** [comment]
@lnielsen Updated and ready from my side.

**Pull request merged**